Paintroid/src/main/java/org/catrobat/paintroid/dialog/JpgInfoDialog.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ * Paintroid: An image manipulation application for Android.
+ * Copyright (C) 2010-2021 The Catrobat Team
+ * (<http://developer.catrobat.org/credits>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.catrobat.paintroid.dialog
+
+import android.app.Dialog
+import android.os.Bundle
+import androidx.appcompat.app.AlertDialog
+import androidx.appcompat.app.AppCompatDialogFragment
+import org.catrobat.paintroid.R
+
+class JpgInfoDialog : AppCompatDialogFragment() {
+    override fun onCreateDialog(savedInstanceState: Bundle?): Dialog =
+        AlertDialog.Builder(requireContext(), R.style.PocketPaintAlertDialog)
+            .setMessage(R.string.pocketpaint_jpg_message_dialog)
+            .setTitle(R.string.pocketpaint_jpg_title_dialog)
+            .setPositiveButton(R.string.pocketpaint_ok) { dialog, which -> dismiss() }
+            .create()
+
+    companion object {
+        @JvmStatic
+        fun newInstance() = JpgInfoDialog()
+    }
```

Comment:
```suggestion
```
That's an old pattern from the deprecated DialogFragment.
Please replace the new instance with the constructor call.

Reply:
Please do it for the other classes too. Thx!

Reply:
done 🙂